### DISCUSSION TITLE
Registration of large amounts of resources and artifacts in DSC v6

### DISCUSSION THREAD

**Discussion by soniasantiago** [International-Data-Spaces-Association/DataspaceConnector#88] [Q&A] [answered]
Hello, 

We are working with the Dataspace Connector v6.0 and wonder whether there is a special mechanism to register large amounts of datasets in the dataspace connector. That is, instead of registering each resource and artifact, one by one, is there a way to register all of them in batch?

In some repository we have seen that you provide some scripts to automatise this process, but we would like to know if there is something more specific for this use case.

Thanks in advance,

Sonia.

**Comment by HeinrichPet** [comment, marked as answer]
Hi,

you can maybe use the [Bootstrapping](https://international-data-spaces-association.github.io/DataspaceConnector/Deployment/Bootstrapping) feature.

Best regards,
Heinrich

**Reply by soniasantiago** [reply]
Thanks for your reply, Heinrich.

Just for the record, a partner working in the same project has made a similar question in the DSC developer's meeting this morning. He's being told that the registration information related to the datasets can also be stored in the connector's local database.
